shell/src/main/java/org/apache/accumulo/shell/commands/MergeCommand.java

```diff
@@ -53,13 +53,7 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       size = ConfigurationTypeHelper.getFixedMemoryAsBytes(cl.getOptionValue(sizeOpt.getOpt()));
     }
     if (startRow == null && endRow == null && size < 0 && !all) {
-      shellState.getWriter().flush();
-      String line = shellState.getReader()
-          .readLine("Merge the entire table { " + tableName + " } into one tablet (yes|no)? ");
-      if (line == null) {
-        return 0;
-      }
-      if (!line.equalsIgnoreCase("y") && !line.equalsIgnoreCase("yes")) {
+      if (!shellState.yorn("Merge the entire table { " + tableName + " } into one tablet")) {
```

Comment:
Not related to your PR, but until no-chop merges are complete, I wonder if this prompt should convey a bit more panic, as in:

`"Are you *REALLY* sure you want to merge the entire table { " + tableName + " } into one tablet?!?!?!"`

Reply:
Updated the prompt wording.